src/commands/dev/index.ts

```diff
@@ -277,6 +281,9 @@ export default class Dev extends BaseCommand {
       default: [],
       sensitive: false,
     }),
+    'secrets-from': Flags.string({
```

Comment:
We should come to a consensus on the flag name. Guessing you missed the slack conv for this ticket. Tyler had proposed `secret-env`, but that means we would need to provide flags for each entity type.

Reply:
My bad. Would you like me to extend the support for the account and cluster secrets as well?

Reply:
Prob worth another conv in engineering. Adding 3 flags feels like a lot. I'd probably be okay with only supporting secrets from environments, but that might not be everyone's expectation.